### ISSUE TITLE
Moved schedule to common schedule repo

### ISSUE THREAD

**Issue by bholt** [uwsampa/grappaBADASSmeeting#1]
We've made it easier to add new schedule pages to the sampa website. I migrated this schedule there: https://github.com/uwsampa/schedule/tree/master/badass

@nelsonje, do you think we can just delete this repo?

**Comment by nelsonje** [comment]
Yes, absolutely! Did you change the deployment path? If so we should add a
redirect, just in case.

**Comment by bholt** [comment]
It now deploys to https://sampa.cs.washington.edu/schedule/badass/15wi.html.

So yeah, we could setup a redirect if you like. I just figured we'd used it a whole 2 times so far, so it probably wouldn't inconvenience anyone too much. This is also easy to find by going to "https://sampa.cs.washington.edu/schedule"

**Comment by nelsonje** [comment]
Well, the link was in a couple emails recently, so I set it up just in
case....

I also had set it up so we didn't need to change it every quarter. I don't
see a value to separating group meeting schedules like this and requiring
someone to do work. But, if you're going to do that work.... :-)

**Comment by jamesbornholt** [comment]
The value is pretty minimal and long-term: we'd like to keep our schedules around for posterity, but wouldn't like to just have one giant list we keep appending stuff to forever.
